presto-druid/src/main/java/com/facebook/presto/druid/segment/V9SegmentIndexSource.java

```diff
@@ -113,6 +111,10 @@ public QueryableIndex loadIndex(List<ColumnHandle> columnHandles)
         columns.put(TIME_COLUMN_NAME, () -> createColumnHolder(TIME_COLUMN_NAME));
 
         Indexed<String> indexed = new ListIndexed<>(availableDimensions);
+
+        // org.joda.time.Interval is used here in the public Druid API and appears in the byte code
+        // even if this loval variable is inlined.
+        Interval dataInterval = Intervals.utc(indexBuffer.getLong(), indexBuffer.getLong());
```

Comment:
you can remove these changes since Interval is still needed by druid.

Reply:
I'd .like to keep the comment so no one else tries to fix this in the future and has to relearn this. at least not until Druid provides a java.time alternative to this constructor.

Reply:
Please add just the comment and don't reorder the code

Reply:
done